app/views/scheduled_tests/index.html.erb

```diff
@@ -0,0 +1,67 @@
+<%- model_class = ScheduledTest -%>
+<div class="page-header">
+  <h1>Your Scheduled Tests</h1>
+</div>
+<div class="sticky sticky_table_header">
+  <table class="table table-heading-table">
+    <thead>
+      <tr>
+        <th class="scheduled-test-last-test-column">Last test</th>
+        <th class="scheduled-test-tests-column">Tests</th>
+        <th class="scheduled-test-next-test-column"><%= model_class.human_attribute_name(:next_test) %></th>
+        <th class="scheduled-test-recurring-column"><%= model_class.human_attribute_name(:recurring) %></th>
+        <th class="scheduled-test-updated-at-column"><%= model_class.human_attribute_name(:updated_at) %></th>
+        <th class="scheduled-test-delete-column"></th>
+      </tr>
+    </thead>
+  </table>
+</div>
+<table class="table table-striped">
+  <tbody>
+    <% @scheduled_tests.each do |scheduled_test| %>
+      <tr class="scheduled-tests-index-row">
+        <td class="scheduled-test-last-test-column cursor-hand"
+            onclick="location.href='<%= edit_scheduled_test_path(scheduled_test) %>'">
+            <% run_status = scheduled_test.last_run_status %>
+          <div class="last-test-status last-test-<%= run_status[:class] %>"><%= run_status[:display] %></div>
+        </td>
+        <td class="scheduled-test-tests-column">
+          <div class="scheduled-test-testsets-container">
+          <% scheduled_test.test_ids.each do |test_id| %>
+            <a href="" class="btn btn-default scheduled-test-testset">
```

Comment:
Clicking the testset buttons doesn't provide any response. Unsure whether this was meant to link through to the test set page or if there was never meant to be any functionality (in which case don't need to format as a button).